pkgs/misc/drivers/pentablet-driver/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, mkDerivation, fetchzip, autoPatchelfHook, libusb1, libX11, libXtst, qtbase, libglvnd }:
+
+mkDerivation rec {
```

Comment:
Replace `mkDerivation` with `stdenv.mkDerivation` and you can drop the `mkDerivation` from the first line

Reply:
no I think when qt is involved, using the mkDerivation from args is correct.

Reply:
Ah, correct. I missed that part.

Reply:
Yep, i replaced it with `wrapQtAppsHook` as this is preffered, at least thats what I've been told on [another PR](https://github.com/NixOS/nixpkgs/pull/84117#issuecomment-617983911).